### PR TITLE
Changes to copy part 1

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR implements the first part of content changes and UX refinements to the copy process.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-766

## Changes in this PR:
- Updated content to the copy page
- Updated review page for copying vacancies
- Cancel copy link that cancels the copy process

### CODE REVIEW

File: app/controllers/hiring_staff/vacancies_controller.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def review
     unless @vacancy.valid?
       redirect_to_incomplete_step
     else
-      state = params[:edit_draft] == 'true' ? 'edit' : 'review'
+      state = params[:edit_draft] == 'true' ? 'edit' :  @vacancy&.state == 'copy' ? 'copy' : 'review'
```
Comment (cesidio) [inline app/controllers/hiring_staff/vacancies_controller.rb:35]: It's not a good practice to have a ternary inside a ternary. Maybe you can move that to a method

File: app/controllers/hiring_staff/vacancies_controller.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def review
...
       @vacancy.update(state: state) unless @vacancy&.state == state
```
Comment (cesidio) [inline app/controllers/hiring_staff/vacancies_controller.rb:36]: I think that if you try to call `update` and you set the same value, nothing happens anyway so you do not need to check that.

File: app/views/hiring_staff/vacancies/vacancy_form_partials/_hidden_state_input.html.haml [hunk 3]
```diff
@@ -1 +1,2 @@
-= f.hidden_field :state, value: @vacancy&.published? ? 'edit_published' : %w(review edit).include?(@vacancy&.state) ? @vacancy.state : 'create'
+- copy = false if local_assigns[:copy].nil?
+= f.hidden_field :state, value: copy ? 'copy' : @vacancy&.published? ? 'edit_published' : %w(copy review edit).include?(@vacancy&.state) ? @vacancy.state : 'create'
```
Comment (cesidio) [inline app/views/hiring_staff/vacancies/vacancy_form_partials/_hidden_state_input.html.haml:2]: Maybe a helper for all this logic, or in the presenter

File: app/controllers/hiring_staff/vacancies_controller.rb (hunk 2 above)
Comment (cpjmcquillan) [reply app/controllers/hiring_staff/vacancies_controller.rb:36]: Ah ok nice, thanks

**Pull request merged**